front/src/modules/simulationResult/components/SpeedSpaceChart/SpeedSpaceSettings.tsx

```diff
@@ -78,7 +79,7 @@ const SpeedSpaceSettings = ({
 
   return (
     <div
-      className={`${showSettings ? 'ml-5' : ''} showSettings`}
+      className={cx({ 'ml-5': showSettings }, 'showSettings')}
```

Comment:
Should we move the boolean class at the end ?

Reply:
Done